utils/extractor/src/index.ts

```diff
@@ -80,7 +82,19 @@ export const Extractor = (
             await asyncBatch(
               capture[sheet.name].data,
               async (chunk) => {
-                await api.records.insert(sheet.id, chunk)
+                const body = zlib.gzipSync(JSON.stringify(chunk))
+                await axios.post(
+                  `${process.env.FLATFILE_API_URL}/v1/sheets/${sheet.id}/records`,
+                  body,
+                  {
+                    headers: {
+                      Authorization: `Bearer ${process.env.FLATFILE_API_KEY}`,
+                      'Content-Type': 'application/json',
+                      'Content-Length': body.length,
+                      'Content-Encoding': 'gzip',
```

Comment:
do we want to allow people to opt out of gzip, for whatever reason

Reply:
IMO the more configuration options we give people the more likely they are to mess it up.